### PR TITLE
Enable forced/cron build every 6 hours

### PR DESCRIPTION
**Pull request by batmat**
To catch issues with possible "upstream" components like plugins or
others that might make master fail.
Goal is to generally catch issues, and more specifically avoid
developers scratching their heads for hours before realizing their
additions have nothing to do with the given failure.

NOTE: expected to fail because of the 4.x release of the `metrics` plugin and the fact we still depend on `master` on the latest version of most plugins.

### CODE REVIEW

**Comment by batmat** [comment]
Let's wait for the merge of #105 and the master be back to green, and this one becomes too at least once before merging.

**Review by mandie722 — APPROVED** [review]

**Pull request merged**
